ags/oidc/authz_flow.py

```diff
@@ -116,6 +116,9 @@ def __init__(self, config):
         self.id_token_max_age = None
         self._keys = {}
 
+    def build_url(base_url, path):
+        return '{base_url}{path}'.format(base_url=base_url, path=path)
+
```

Comment:
This should be a method. Also, since `base_url` is always `self.broker_url`, you shouldn't need to pass it in, e.g.:
```python
def build_url(self, path):
    return '{base_url}{path}'.format(base_url=self.broker_url, path=path)

# and later

return self.build_url(self._auth_endpoint)
```